bots/minecraft.bot.js

```diff
@@ -0,0 +1,69 @@
+const _ = require('lodash');
+const Config = require('pico-conf');
+const Slack = require('pico-slack');
+const RCon = require('rcon');
+
+const ASKING_FOR_NOW = /(?:any\s*one|any\s*body).*(?:on|playing).*/i;
+const ASKING_FOR_LATER = /(?:when|any\s*one|any\s*body).*(?:planning|going|thinking).*(?:be on|playing|hopping on).*/i;
+
+Slack.onMessage(async (msg) => {
+	if (msg.channel != 'minecraft' && !msg.isDirect) {
+		return;
+	}
+
+	const whosOnlineNow = msg.text.match(ASKING_FOR_NOW);
+	const whosOnlineLater = msg.text.match(ASKING_FOR_LATER);
+
+	if (!whosOnlineNow && !whosOnlineLater) {
+		return;
+	}
+
+	Slack.react(msg, 'thinking_face_b');
+
+	try {
+		const reply = await makeReply(whosOnlineNow, whosOnlineLater);
+		Slack.sendAs('minecraft', 'creeper', msg.channel_id, reply);
+	} catch (err) {
+		Slack.error(err);
+	}
+});
+
+const makeReply = async (whosOnlineNow, whosOnlineLater) => {
+	const players = await fetchPlayers();
```

Comment:
Thoughts on `playerNames = await fetchPlayerNames()` to make it clear that it's an array of strings as opposed to some sort of player object?

Reply:
Usually if I'm writing code that doesn't _have_ the concept of a (player) object, I prefer to keep the names tighter. I'm happy to change it though if you still think the ambiguity is a bit too much.